solution/src/main/openapi/solution.yaml

```diff
@@ -1104,10 +1180,47 @@ components:
           description: the Parameter Group id
           minLength: 1
           maxLength: 50
+        description:
+          type: string
+          description: A description of the parameter group
+        labels:
+          $ref: '#/components/schemas/TranslatedLabels'
+        isTable:
+          type: boolean
+          description: Does the group define a table
+        options:
+          type: object
+          description: Freeform options
+          additionalProperties: true
+        parentId:
+          type: string
+          description: The Run Template Group parent Id
+        parameters:
+          type: array
+          description: An ordered list of Run Template Parameters
+          items:
+            type: string
+            description: A Run Template Parameter id
+      required:
```

Comment:
Should we have more required fields here (or default values to make them non-optional)?
Like `description`, `isTable`, `parameters`, more?

Reply:
I set a default value for :

- isTable (to false)
- parameters (to [] )